rest/admin_api.go

```diff
@@ -181,7 +181,14 @@ func getAuthScopeHandleCreateDB(ctx context.Context, h *handler) (bucketName str
 		Bucket string `json:"bucket"`
 	}
 	reader := bytes.NewReader(bodyJSON)
-	err = DecodeAndSanitiseConfig(ctx, reader, &dbConfigBody, false)
+
+	// read and decode json to pull bucket name from the config
```

Comment:
I think you can call `readSantizedJson` here.

The thing that might happen is that you need to call `ConvertBackQuotedStrings` to make it valid json. The test should include a config with backquotes